tests/test_herbstclient.py

```diff
@@ -414,3 +414,18 @@ def reply_output():
     assert re.search(f'could not get window property "{faulty_name}"', hc.reply.stderr)
     assert not re.search('EEE', hc.reply.stderr)
     assert hc.reply.returncode == 1
+
+
+def test_command_tokenization(hlwm):
```

Comment:
The test is related, because it ensures that it's `<=` in `while (offset <= count)`. Maybe `tokenization` isn't the best word for it, because it's not about tokenization on the command line but about tokenization when reading the x11-property (there's no command line tokenization at all by the way)